src/main/lrsql/ops/query/generate.clj

```diff
@@ -0,0 +1,16 @@
+(ns lrsql.ops.query.generate
+  (:require [clojure.spec.alpha :as s]
+            [clojure.walk :as walk]
+            [lrsql.spec.reaction :as react-spec]))
+
+(s/fdef generate
+  :args (s/cat :cond-map (s/map-of :react-spec/condition-name
+                         :react-spec/val)
+               :statement map?))
+
+(defn generate [cond->statement template]
```

Comment:
Don't know if `generate` is a great name for this, it's a little too general. Maybe something like generate-statement if you move it to that reactions util NS

Reply:
renamed to `generate-statement`